src/components/Table/TableHead/_table-head.scss

```diff
@@ -91,15 +105,21 @@
   padding: 0 $spacing-05;
 }
 
-.#{$iot-prefix}--table-head--table-header {
-  .#{$prefix}--table-header-label {
-    min-width: var(--table-header-width);
-    max-width: var(--table-header-width);
-    white-space: nowrap;
-    overflow-x: hidden;
-    overflow-y: hidden;
-    text-overflow: ellipsis;
+.#{$iot-prefix}--table-head {
+  &--overflow {
+    flex-shrink: 0;
   }
 
-  vertical-align: middle;
+  &--table-header {
+    .#{$prefix}--table-header-label {
+      height: 100%;
```

Comment:
Why was this height added? It seems to be the cause of this misalignment:
![image](https://user-images.githubusercontent.com/5167091/93315897-dd56bf80-f80b-11ea-92ad-aa4cd3f73c35.png)

Reply:
a fair amount of these issues popped up when I merged in master. I will resolve the issues that were fixed from that.